### PR TITLE
#43 - Add interfaces for common models of Shopware 6

### PR DESCRIPTION
**Pull request by rmakara**
**Changes:**
* Add interfaces for `common` group.
* Right now it contains following interfaces: `Entity`, `Collection`, `CustomField`.

**To do:**
* `CustomField` interface to be filled in #61 

**Issue:**
#43

### CODE REVIEW

File: packages/shopware-6-client/src/interfaces/models/common/Entity.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export interface Entity {
+  _uniqueIdentifier: string;
+  versionId: string;
+  translated: [];
```
Comment (patzick) [inline packages/shopware-6-client/src/interfaces/models/common/Entity.ts]: Which type of Array do we have here?

File: packages/shopware-6-client/src/interfaces/models/common/Collection.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export interface Collection {
+  elements: [];
```
Comment (patzick) [inline packages/shopware-6-client/src/interfaces/models/common/Collection.ts]: please provide an example of this collection usage

also, we need to add a type of elements array

File: packages/shopware-6-client/src/interfaces/models/common/Entity.ts (hunk 1 above)
Comment (rmakara) [reply packages/shopware-6-client/src/interfaces/models/common/Entity.ts]: Just an array in Shopware. No specific type.

File: packages/shopware-6-client/src/interfaces/models/common/Collection.ts (hunk 2 above)
Comment (rmakara) [reply packages/shopware-6-client/src/interfaces/models/common/Collection.ts]: We can delete this one as we deleted all the interfaces suffixed with Collection.

**Comment by rmakara** [comment]
@patzick suggestions applied. :)

**Pull request merged**